src/webportal/src/app/job-submission/utils/conn.js

```diff
@@ -52,16 +52,19 @@ export async function listUserVirtualClusters(user) {
   return get(userInfo, 'virtualCluster', []);
 }
 
-export async function listHivedSkuTypes() {
+export async function listHivedSkuTypes(virtualCluster) {
   if (config.launcherScheduler !== 'hivedscheduler') {
     return {};
   }
   return wrapper(async () =>
-    (await fetch(`${config.restServerUri}/api/v2/cluster/sku-types`, {
```

Comment:
better use url-join to avoid ending '/' bugs
[example](https://github.com/microsoft/pai/blob/f2c6f06944bb703923dd134d08532d608561f3ac/src/webportal/src/app/home/home/conn.js#L39)